pages/challenge.js

```diff
@@ -12,6 +12,32 @@ const RepoLink = 'https://github.com/OperationCode/front-end/';
 const ChallengeLink = `${RepoLink}blob/main/pages/challenge.js`;
 const CompareLink = `${RepoLink}compare`;
 
+export const MakeColumnsFromArray = ({ ary }) => {
+  const numberNamesPerColumn = 20;
+  const numberCols = Math.floor(ary.length / numberNamesPerColumn) + 1;
+
+  const content = [];
+  let col = 0;
+
+  while (col < numberCols) {
+    const colPersonStartIndex = col * numberNamesPerColumn;
+    const colPersonEndIndex = col * numberNamesPerColumn + numberNamesPerColumn;
+    const elements = challengers.slice(colPersonStartIndex, colPersonEndIndex);
+
+    content.push(
+      <div key={col} className={styles.challengerListColumn}>
```

Comment:
this should be `ol`. `li` can only ever be children of `ol` elements and `ol` element should only ever have `li` elements as immediate children.

See here for more: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ol